src/MenuInventory.cpp

```diff
@@ -83,17 +85,52 @@ MenuInventory::MenuInventory()
 				Point pos = Parse::toPoint(infile.val);
 				closeButton->setBasePos(pos.x, pos.y, Utils::ALIGN_TOPLEFT);
 			}
-			// @ATTR equipment_slot|repeatable(int, int, string) : X, Y, Slot Type|Position and item type of an equipment slot.
+			// @ATTR set_button|int, int, filename : Widget X, Widget Y, Image file|Position and image filename for an equipment swap set button.
```

Comment:
I think the `set_button` property should include the set id by value instead of inferring it from parsing order. That way, it will be possible for modders to redefine the button graphics without creating more sets.

Reply:
Now it's fixed with this commit: ef5368e